templates/zerver/help/import-from-slack.md

```diff
@@ -1,149 +1,293 @@
+|-------------------------|---------------|
+| Workspace Primary Owner | Owner         |
+| Workspace Owner         | Owner         |
+| Workspace Admin         | Administrator |
+| Member                  | Member        |
+| Single Channel Guest    | Guest         |
+| Multi Channel Guest     | Guest         |
+| Channel creator         | none          |
 
-- Slack doesn't export private channels or direct messages unless you pay
-  for Slack Plus or contact Slack support. See
-  [Slack's documentation](https://get.slack.help/hc/en-us/articles/204897248-Guide-to-Slack-import-and-export-tools)
-  for more details.
+- Messages in threads are imported, but they are not explicitly marked as
+  being in a thread.
 
-- (Slack Plus import) Message edit history is currently not imported.
+- Message edit history and `@user joined #channel_name` messages are not imported.
 
-- Slack doesn't export user settings or organization settings, so
-  you'll need to configure these manually.
+### Clean up after the Slack export
 
-- Import of [user roles](/help/roles-and-permissions):
-    - Slack's `Workspace Primary Owner` and `Workspace Owner` users
-    are mapped to Zulip `Organization owner` users.
-    - Slack's `Workspace Admin` users are mapped to Zulip's `Organization
-      administrator` users.
-    - Slack's `Member` users is mapped to Zulip `Member` users.
-    - Slack's `Single Channel Guest` and `Multi Channel Guest` users
-    are mapped to Zulip `Guest` users.
-    - Slack's `Channel creators` have no special permissions in Zulip.
+Once your organization has been successfully imported in to Zulip, you should
+delete [the Slack app](https://api.slack.com/apps) that you created.  This will
+prevent the OAuth token from being used to access your Slack workspace in the
+future.
 
-- The "joined #channel_name" messages are not imported.
+## Get your organization members started
+
+Once the import completes, all users from your Slack organization will have
+accounts in your new Zulip organization. You will need to:
+
+{start_tabs}
+
+1. Decide how users will log in for the first time (see below).
+
+1. Share the URL for your new Zulip organization, and (recommended) the [Getting
+   started with Zulip guide](/help/getting-started-with-zulip).
+
+{end_tabs}
+
+
+### Decide how users will log in
+
+When user accounts are imported from Slack, users initially do not have passwords
+configured. There are a few options for how users can log in for the first time.
+
+!!! tip ""
+    For security reasons, Slack passwords are never exported.
+
+### Allow users to log in with non-password authentication
+
+When you create your organization, users will immediately be able to log in with [
+authentication methods](help/configure-authentication-methods) that do not require a password.
+Authentication with Google, GitHub, GitLab, and Apple is enabled by default, and
+you can set up other authentication methods such as LDAP or [SAML](/help/saml-authentication).
```

Comment:
This paragraph seems to be talking about Cloud specifically (due to the "Authentication with Google, GitHub, GitLab, and Apple is enabled by default" sentence) - so mentioning LDAP doesn't quite fit with that, because we don't offer LDAP on Cloud

Reply:
Hm, what would be the best way for the sentence to work for self-hosters too? Maybe something like this?

> Zulip offers a variety of authentication methods, including Google, GitHub, GitLab, Apple, LDAP and [SAML](/help/saml-authentication).

Reply:
Yeah, that seems like a good way to cover it.